electron/execution.ts

```diff
@@ -293,21 +304,19 @@ async function onSetMode(mode) {
   return selector;
 }
 
-async function onLinkExternal(url) {
+async function onLinkExternal(url: string) {
   try {
     await shell.openExternal(url);
   } catch (e) {
     logger.error(e);
   }
 }
 
-function setupListeners() {
-  ipc.answerRenderer('record-journey', recordJourneys);
-  ipc.answerRenderer('run-journey', onTest);
+export default function setupListeners() {
+  ipc.answerRenderer<{ url: string }>('record-journey', recordJourneys);
+  ipc.answerRenderer<RunJourneyOptions>('run-journey', onTest);
   ipc.answerRenderer('save-file', onFileSave);
-  ipc.answerRenderer('actions-to-code', onTransformCode);
-  ipc.answerRenderer('set-mode', onSetMode);
-  ipc.answerRenderer('link-to-external', onLinkExternal);
+  ipc.answerRenderer<{ actions: Steps; isSuite: boolean }>('actions-to-code', onTransformCode);
```

Comment:
```suggestion
  ipc.answerRenderer<TransformCodeOptions>('actions-to-code', onTransformCode);
```

Same question about type.

Reply:
Yeah I agree. I made the type definition for all non-primitive arguments 👍